docs/extras/feature-obfuscation/feature-obfuscation.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Feature Obfuscation
+sidebar_position: 1
+---
+
+Feature flags are often used to hide upcoming features before release. Normally, not showing the feature on a UI
+is enough to conceal it from users. However, in some cases it may be important to ensure that no trace of the feature
```

Comment:
```suggestion
is enough to conceal it from users. However, in some cases, it may be important to ensure that no trace of the feature
```

Reply:
I kinda prefer it the original way :O that's how I would say it out loud